### PR TITLE
Explicitly mention alpha status of snapshot APIs.

### PR DESCRIPTION
**Pull request by draghuram**
Also added a section about creating volumes from snapshots.

### CODE REVIEW

**Comment by draghuram** [comment]
@xing-yang, This PR covert part of what we discussed in:
    https://groups.google.com/forum/#!msg/kubernetes-sig-storage/EBGjZJ-0gGs/0wWhZMgXAgAJ

File: content/en/docs/concepts/storage/volume-snapshots.md [hunk 1]
```diff
@@ -129,4 +130,12 @@ using the attribute `snapshotClassName`.
 Only VolumeSnapshotContents of the requested class, ones with the same `snapshotClassName`
 as the VolumeSnapshot, can be bound to the VolumeSnapshot.
 
+## Provisioning Volumes from Snapshots
+
+A new volume, pre-populated with data from a snapshot, can be provisioned using
+*dataSource* field in the `PersistentVolumeClaim` object.
```
Comment (steveperry-53) [inline content/en/docs/concepts/storage/volume-snapshots.md]: Add "the".'
using the *dataSource* field.

Remove "please" to align with our [style guide](https://kubernetes.io/docs/contribute/style/style-guide/).
For more details, see ...

File: content/en/docs/concepts/storage/volume-snapshots.md [hunk 2]
```diff
@@ -129,4 +130,12 @@ using the attribute `snapshotClassName`.
 Only VolumeSnapshotContents of the requested class, ones with the same `snapshotClassName`
 as the VolumeSnapshot, can be bound to the VolumeSnapshot.
 
+## Provisioning Volumes from Snapshots
+
+A new volume, pre-populated with data from a snapshot, can be provisioned using
```
Comment (sftim) [inline content/en/docs/concepts/storage/volume-snapshots.md]:
```suggestion
You can create a new Volume based on a snapshot. To request this, set the
```

File: content/en/docs/concepts/storage/volume-snapshots.md [hunk 3]
```diff
@@ -129,4 +130,12 @@ using the attribute `snapshotClassName`.
...
+*dataSource* field in the `PersistentVolumeClaim` object.
+
+For more details, please see
+[here](/docs/concepts/storage/persistent-volumes/#volume-snapshot-and-restore-volume-from-snapshot-support).
```
Comment (sftim) [inline content/en/docs/concepts/storage/volume-snapshots.md]:
```suggestion
[Volume Snapshot and Restore Volume from Snapshot](/docs/concepts/storage/persistent-volumes/#volume-snapshot-and-restore-volume-from-snapshot-support).
```

File: content/en/docs/concepts/storage/volume-snapshots.md (hunk 1 above)
Comment (steveperry-53) [inline content/en/docs/concepts/storage/volume-snapshots.md]: Our style guide recommends active voice.
Suggestion:
You can provision a new volume, pre-populated with data from a snapshot, by using the *dataSource* field ...

**Pull request merged**